src/en/getting-started.md

```diff
@@ -1,255 +1,323 @@
+inspector on the left will disappear, to show that everything is working as
+expected.
+
+!!! Tip: You can check and manage existing models through the JAAS GUI by
+clicking on your username
+
+## Use the command line
+
+Juju can also be used from the command line. Models you've created in JAAS are
+also able to be operated against via the CLI and vice versa. In order to use
+the command line, you will first need to install the Juju client software on
+your machine.
+
+Juju is available for various types of Linux, macOS, and Windows.
+Click on the sections below for the relevant instructions or visit the
+[install instructions][install] for a thorough run down of your options.
+
+To most quickly get started pick your OS from below.
 
-This will fetch a 'bundle' from the Juju store. A bundle is a pre-packaged set
-of applications, in this case 'MediaWiki', and a database to run it 
-with. Juju will install both applications and add a relation between them - 
-this is part of the magic of Juju: it isn't just about deploying software, Juju 
-also knows how to connect it all together.
+<style>
+details  {
+    padding-bottom: 6px;
+}
+</style>
+
+^# On an OS which supports snap packages
+
+   For Ubuntu 16.04 LTS (Xenial) and other OSes which have snaps enabled
+   ([see the snapcraft.io site][snapcraft]) you can install the latest
+   stable release of Juju with the command:
+
+         sudo snap install juju --classic
+
+   You can check which version of Juju you have installed with
+
+         sudo snap list juju
+
+   And update it if necessary with:
+
+         sudo snap refresh juju
+
+   It is possible to install other versions, including beta releases of
+   Juju via a snap package. See the [install][install] for more information.
+
+
+^# From the Ubuntu PPA
+
+   A PPA resource is available containing the latest stable version of
+   Juju. To install:
+
+       sudo add-apt-repository ppa:juju/stable
+       sudo apt update
+       sudo apt install juju
+
+^# For Windows
+
+   A Windows installer is available for Juju.
+
+   [juju-setup-2.1.2.exe](https://launchpad.net/juju/2.1/2.1.2/+download/juju-setup-2.1.2.exe)([md5](https://launchpad.net/juju/2.1/2.1.2/+download/juju-setup-2.1.2.exe/+md5))
+
+   Please see the [install docs][install] for other
+   versions.
+
+^# For macOS
+
+   The easiest way to install Juju on macOS is with the brew package
+   manager. With brew installed, simply enter the following into a
+   terminal:
+
+       brew install juju
+
+   If you previously installed Juju with brew, the package can be
+   updated with the following:
+
+       brew upgrade juju
+
+   For alternative installation methods, see the [install docs][install].
+
+^# For CentOS
+
+   A pre-compiled binary is available for CentOS.
+
+   Please see the [install docs][install] for a link to the latest
+   version.
+
+
+## Register or login to JAAS
+
+To connect to JAAS from the command line you'll need to login. Enter the following command:
```

Comment:
on the command line, _login_ could be ambiguous. How about `authenticate the CLI against JAAS with the following command:`